packages/manager/modules/vps/src/dashboard/tile/configuration/index.html

```diff
@@ -146,6 +169,16 @@
                     ></span>
                 </a>
             </p>
+            <oui-message
+                type="info"
+                data-ng-if="$ctrl.serviceInfos.contactAdmin !== $ctrl.nichandle"
+            >
+                <p
+                    class="mb-0"
+                    data-translate="vps_dashboard_tile_configuration_upgrade_wrong_nic_storage"
+                    data-translate-values="{ nichandle: '<strong>' + $ctrl.serviceInfos.contactAdmin + '</strong>' }"
+                ></p>
+            </oui-message>
```

Comment:
Related to this:
https://github.com/ovh/manager/pull/8919/files#r1109809369

Reply:
Maybe in second time, the first priority is unblock the tech customer